DEV-2815 (#844)
cloudposse/atmos: website/docs/cli/commands/version.mdx

atmos version
```

This will show the CLI version. 

From time to time, Atmos will check for updates. The frequency is configured in the `atmos.yaml`. There are six frequency options: minute, hourly, daily, weekly, monthly, and yearly. The default is to check `daily`, and if any unsupported values are passed this default will be used.

> It is also possible to turn off version checks in `atmos.yaml` by switching `enable` to `false`, or by setting the `ATMOS_VERSION_CHECK_ENABLED` environment variable to `false`, which overriders the config's settings. 

Reviewer: ```suggestion
It is also possible to turn off version checks in `atmos.yaml` by switching `enable` to `false`, or by setting the `ATMOS_VERSION_CHECK_ENABLED` environment variable to `false`, which overrides the config's settings. 
```

Author: @osterman , thanks, I also added a logic description forr suffixes etc, please check the wording if its ok

```shell
atmos version --check
```

The version command supports a `--check` flag. 

> It will force atmos to check for a new version, irrespective of the configuration settings.  

Reviewer: ```suggestion
This will force Atmos to check for a new version, irrespective of the configuration settings.  
```
```suggestion
It will force atmos to check for a new version, irrespective of the configuration settings.  
```

Author: Done

atmos version
```

> This will show the CLI version. 
> The frequency with which the update box message in version is shown is configured in atmos.yaml.
> There are two options daily and weekly. The default value is set to daily. 
> It is also possible to turn it off in atmos.yaml file by switching enable to false. 
> Or set an env variable ATMOS_VERSION_CHECK_ENABLED to false, 
> this overriders the config's settting. 

Reviewer: ```suggestion
This will show the CLI version. 

From time to time, Atmos will check for updates. The frequency is configured in the `atmos.yaml`. There are two options `daily` and `weekly`. The default is to check `daily`, and if any unsupported values are passed this default will be used.

It is also possible to turn off version checks in `atmos.yaml` by switching `enable` to `false`, or by setting the `ATMOS_VERSION_CHECK_ENABLED` environment variable to `false`, which overriders the config's settings. 
```

```shell
atmos version --check
```

> The version command supports --check flag. 

Reviewer: ```suggestion
The version command supports a `--check` flag. 
```

> It will show the update box message irrespectively of the config settings.  

Reviewer: ```suggestion
It will force atmos to check for a new version, irrespective of the configuration settings.  
```

atmos version
```

This will show the CLI version. 

> From time to time, Atmos will check for updates. The frequency of these checks is configured in the atmos.yaml file.

Reviewer: ```suggestion
From time to time, Atmos will check for updates. The frequency of these checks is configured in the `atmos.yaml` file.
```

Atmos supports three ways to specify the update check frequency:

1. As an integer: Specify the number of seconds between checks (for example, 3600 for hourly checks).

> 2. As a duration with a suffix: Use a time suffix to indicate the interval (for example, 1m for one minute, 5h for five hours, or 2d for two days).

Reviewer: ```suggestion
2. As a duration with a suffix: Use a time suffix to indicate the interval (for example, `1m` for one minute, `5h` for five hours, or `2d` for two days).
```
